### DISCUSSION TITLE
I made a simplified tutorial using function components

### DISCUSSION THREAD

**Discussion by jimmythecode** [gilbarbara/react-joyride#666] [Show and tell]
I made this tutorial on sandbox:
https://codesandbox.io/s/joyride-basic-demos-v6z32?file=/src/App.js

I hope it'll be useful for anyone like me, who isn't as familiar with React classes, typescript, or other features/libraries in the existing tutorial.

I'm going to do another one on controlled components next, as I've been struggling with that.

**Comment by gilbarbara** [comment]
Your sandbox is broken

**Reply by jimmythecode** [reply]
Well that's very frustrating. It seems you can't turn around for a moment before looking back to find Sandbox has deleted random bits of code. Hopefully this works:
[Sandbox](https://codesandbox.io/s/joyride-basic-demos-v6z32?file=/src/App.js)

**Reply by jimmythecode** [reply]
Oh, also I can't get the sandbox with your examples to work either: 
https://codesandbox.io/s/github/gilbarbara/react-joyride-demo

**Reply by gilbarbara** [reply]
It seems like an error from the codesandbox installer. I haven't changed the demo for a couple of months and it was working.
I've contacted them already.

**Comment by ayozebarrera** [comment]
Hi, you are having the same problem as me.
react-joyride is duplicating elements on the body and it doesn't remove then when stop it!

![Captura de pantalla 2021-04-14 a las 9 15 59](https://user-images.githubusercontent.com/6180188/114678303-063c2600-9d03-11eb-9ac3-d23596ea3ef9.png)

Trying to figure out why =( if you see something tell me!
